src/include/sof/drivers/interrupt.h

```diff
@@ -163,4 +163,12 @@ static inline void interrupt_global_enable(uint32_t flags)
 	arch_interrupt_global_enable(flags);
 }
 
+/* disables all IRQ sources on current core */
+#define local_irq_save(flags) \
```

Comment:
why not `static inline`?

Reply:
I would need to assign flags everywhere then.